docs/src/props-table.js

```diff
@@ -70,8 +89,8 @@ function BasePropRows({passthroughPropsLink, elementType, isPolymorphic, refType
 function PassthroughPropsRow({elementName, isPolymorphic, passthroughPropsLink}) {
   return (
     <tr>
-      <Box as="td" colSpan={4} fontSize={1} verticalAlign="top" fontStyle="italic">
-        Additional props are passed through to the <InlineCode>&lt;{elementName}&gt;</InlineCode> element. See{' '}
+      <Box as="td" colSpan={4} fontSize={1} verticalAlign="top">
+        Additional props are passed to the <InlineCode>&lt;{elementName}&gt;</InlineCode> element. See{' '}
```

Comment:
What do you think about adding something like a ⚠️ to call out this row?

Reply:
Hmm, not sure. ⚠️ makes me think is a "bad" thing but I'm not sure it is 🤷

Reply:
I think that's fair. 👍🏻